src/fs/bundle.rs

```diff
@@ -0,0 +1,235 @@
+use crate::fs::{FsNode, GuestPath};
```

Comment:
Oh right, can you add the MPL-2.0 license header here, and a doc comment (a one-liner is fine) summarising what this file is for? The license header is necessary because MPL-2.0 is a per-file rather than whole-project license, so it needs to be explicit which files it applies to, and the doc comments are just a thing I like to have to make the project nice to navigate in `cargo doc`.